src/repositories/northstar.js

```diff
@@ -140,6 +140,39 @@ export const updateEmailSubscriptionTopics = async (
   return null;
 };
 
+/**
+ * 
+ * @param {String} id 
+ * @param {[EmailSubscriptionStatus]} emailSubscriptionStatus
+ * @param {Boolean} subscribed 
+ * @param {Object} options 
+ */
+export const updateEmailSubscriptionStatus = async (
+  id,
+  emailSubscriptionStatus,
+  options,
+) => {
+  logger.debug('Updating email subscription status for user in Northstar', {
+    id,
+    emailSubscriptionStatus,
+  });
+  try {
+    const response = await updateUser(
+      id, {
+        email_Subscription_Status: emailSubscriptionStatus.subscribed
```

Comment:
I think we just want to pass `emailSubscriptionStatus` here so like:
```suggestion
        email_Subscription_Status: emailSubscriptionStatus
```

Reply:
still doesn't work 😕but no worries, we'll figure this out.